sktime/transformations/panel/dictionary_based/_sfa.py

```diff
@@ -118,7 +118,6 @@ class SFA(BaseTransformer):
         # testing configuration
         # ---------------------
         # SFA is non-compliant with any transformer interfaces, #2064
-        "tests:skip_all": True,
```

Comment:
this change still remains to be put back

Reply:
no problem! As already said, we intentionally did pick this task that has a relatively simple scope in python, since it helps getting familiar with GitHub! I hope I did not promise too much 😃 

Let us know if you need any help with "how to do x or y". (discord/dev-chat)